docker/configs/grafana/grafana.ini

```diff
@@ -0,0 +1,705 @@
+    ##################### Grafana Configuration Example #####################
+    #
+    # Everything has defaults so you only need to uncomment things you want to
+    # change
+
+    # possible values : production, development
+    app_mode = production
+
+    # instance name, defaults to HOSTNAME environment variable value or hostname if HOSTNAME var is empty
+    ;instance_name = ${HOSTNAME}
+
+    #################################### Paths ####################################
+    [paths]
+    # Path to where grafana can store temp files, sessions, and the sqlite3 db (if that is used)
+    ;data = /var/lib/grafana
+
+    # Temporary files in `data` directory older than given duration will be removed
+    ;temp_data_lifetime = 24h
+
+    # Directory where grafana can store logs
+    ;logs = /var/log/grafana
+
+    # Directory where grafana will automatically scan and look for plugins
+    ;plugins = /var/lib/grafana/plugins
+
+    # folder that contains provisioning config files that grafana will apply on startup and while running.
+    ;provisioning = conf/provisioning
+
+    #################################### Server ####################################
+    [server]
+    # Protocol (http, https, h2, socket)
+    ;protocol = http
+
+    # The ip address to bind to, empty will bind to all interfaces
+    ;http_addr =
+
+    # The http port  to use
+    ;http_port = 3000
+
+    # The public facing domain name used to access grafana from a browser
+    ; domain = localhost
+
+    # Redirect to correct domain if host header does not match domain
+    # Prevents DNS rebinding attacks
+    ;enforce_domain = false
+
+    # The full public facing url you use in browser, used for redirects and emails
+    # If you use reverse proxy and sub path specify full url (with sub path)
+    ; root_url = https://%(domain)s
+    ; root_url = %(protocol)s://%(domain)s:%(http_port)s/
+
+
+    # Serve Grafana from subpath specified in `root_url` setting. By default it is set to `false` for compatibility reasons.
+    ;serve_from_sub_path = false
+
+    # Log web requests
+    ;router_logging = false
+
+    # the path relative working path
+    ;static_root_path = public
+
+    # enable gzip
+    enable_gzip = true
+
+    # https certs & key file
+    ;cert_file =
+    ;cert_key =
```

Comment:
Since Grafana has https certificate options build in we might want to apply them right away so that it is protected. It's surely not a service to expose beyond the local net, but since the other services are secured, this UI should be as well...